execution-engine/src/wasi/strace.rs

```diff
@@ -42,23 +111,42 @@ impl Strace {
         _buf_len: u32,
         _result_ptr: u32,
     ) {
+        if self.arg() {
+            return;
+        }
         // NOT YET IMPLEMENTED
+        eprint!("DIRENTS")
     }
 
     pub fn arg_events<T: MemoryHandler>(&mut self, _mem: &mut T, _events: u32, _size: u32) {
+        if self.arg() {
+            return;
+        }
         // NOT YET IMPLEMENTED
```

Comment:
Are these going to be implemented before the PR is merged?

Reply:
Probably not?

Reply:
But are they tracked in the issue tracker, at least?

Reply:
Yes, it's #303.